### PR TITLE
Add 3.13 folder and update nightly

### PR DESCRIPTION
**Pull request by ColeHiggins2**

### CODE REVIEW

File: _includes/manuals/nightly/3.2.2_installer_options.md [hunk 1]
```diff
@@ -1389,282 +1397,2490 @@ kafo-export-params -c /etc/foreman-installer/scenarios.d/foreman.yaml -f html
...
+        <td>proxy feature listens on http, https, or both</td>
+      </tr>
+      <tr>
+        <td style='white-space:nowrap'>--foreman-proxy-plugin-shellhooks-version</td>
+        <td>plugin package version, it&#39;s passed to ensure parameter of package resource can be set to specific version number, &#39;latest&#39;, &#39;present&#39; etc.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+  <td style='white-space:nowrap'>--puppet-dir-group</td>
```
Comment (ColeHiggins2) [inline _includes/manuals/nightly/3.2.2_installer_options.md]: @ekohl Is this supposed to be separated from the table?

File: _includes/manuals/nightly/3.2.2_installer_options.md [hunk 2]
```diff
@@ -1389,282 +1397,2490 @@ kafo-export-params -c /etc/foreman-installer/scenarios.d/foreman.yaml -f html
...
       </tr>
       <tr>
         <td style='white-space:nowrap'>--puppet-dir</td>
         <td>Override the puppet directory.</td>
       </tr>
       <tr>
-        <td style='white-space:nowrap'>--puppet-dir-group</td>
-        <td>Group of the base puppet directory, used when puppet::server is false.</td>
+      <div id="all-installer-options">
+  <table class="table table-bordered table-condensed">
```
Comment (ekohl) [inline _includes/manuals/nightly/3.2.2_installer_options.md]: This looks odd here. Was the text inserted twice?

**Review by ekohl — COMMENTED** [review]
Given the difficulty with this, perhaps we should first merge https://github.com/theforeman/theforeman.org/pull/2194 and rebase on top of it?

File: _includes/manuals/nightly/3.2.2_installer_options.md (hunk 2 above)
Comment (ColeHiggins2) [reply _includes/manuals/nightly/3.2.2_installer_options.md]: I just copied the output and replaced what was there... So what we see is what was generated

**Comment by ekohl** [comment]
https://github.com/theforeman/theforeman.org/pull/2194 was merged, so there's now a merge conflict.

File: _includes/manuals/nightly/3.2.2_installer_options-params.md
Comment (ekohl) [inline _includes/manuals/nightly/3.2.2_installer_options-params.md:1]: @evgeni this is quoting a lot more than I expected. It's not wrong, but really shows there were many (potential) issues before.

File: _includes/manuals/3.13/3.2.2_installer_options.md [hunk 3]
```diff
@@ -0,0 +1,2547 @@
...
+{% endhighlight %}
+
+#### Plugins and compute resources
+
+The installer contains a number of high level modules (e.g. "foreman", "puppet") and additionally a number of smaller modules for additional functionality, such as plugins and compute resource support.  These can be added with the "--enable" switches, or the default options can be disabled with "--no-enable" switches.
+
+More information about compute resources can be found in the [Compute Resources section](/manuals/{{page.version}}/index.html#5.2ComputeResources) and plugins in the [Plugins section](/manuals/{{page.version}}/index.html#6.Plugins).
+
+<!-- Generated based on `foreman-installer -h | grep enable-` -->
+<div>
```
Comment (ekohl) [inline _includes/manuals/3.13/3.2.2_installer_options.md]: Please make sure this is equal to nightly

**Review by ekohl — CHANGES_REQUESTED** [review]

File: _includes/manuals/3.13/3.2.2_installer_options.md (hunk 3 above)
Comment (ColeHiggins2) [reply _includes/manuals/3.13/3.2.2_installer_options.md]: The script didn't do this... I had to manually create these files to match nightly

**Review by ekohl — APPROVED** [review]
Thanks! Merge whenever you think it's a good time.

**Pull request merged**